### ISSUE TITLE
FonAwesomeIcons class: Use const instead of readonly + missing icons

### ISSUE THREAD

**Issue by f4n0** [Megabit/Blazorise#4774]
**Is your feature request related to a problem? Please describe.**
Hi, I would like to use the fontawesome icons, but as of now, my code uses Attributes and so I cannot use the class FontAwesomeIcons. 

see this screenshot
![image](https://github.com/Megabit/Blazorise/assets/36199751/3a4215be-ca42-4daa-b6a5-8ad552d24ee5)

**Describe the solution you'd like**
Is it possible to move them as const?  this will solve this problem!

### Additional context
Then, there are icons that are missing (eg. Docker icon)

**Comment by stsrki** [comment]
What is that `Icon` property that you try to assign?

**Comment by David-Moreira** [comment]
It's an `Attribute`, something like `[Attribute(Icon="MyIcon")]`.
I agree with @f4n0 I don't think there's a reason to use readonly here. Since we just use it as a constant anyway. I would agree with moving it to constant, @stsrki ?

As for the time being, @f4n0 you can always just create your own mapping and make them constant. Or if it's just a few places, just point to the font awesome class. Those static helpers only really point to the fa classes in this format: "fa-equals", etc...

**Comment by f4n0** [comment]
Yes, as of now I've created my own mapping, but I suppose it will be better to use yours (as it has all the updates).
But yes, is it not urgent, it is just a nice-to-have!
